library/Rules/Lowercase.php

```diff
@@ -13,10 +13,29 @@
 
 namespace Respect\Validation\Rules;
 
-class Lowercase extends AbstractRule
+use function is_string;
+use function mb_detect_encoding;
+use function mb_strtolower;
```

Comment:
@Respect/troubleshooters do we have some rules about declare use of php functions like this? I don't do this on my commits and I don't see this in the code. I just don't know if is good to keep or not.

Reply:
I prefer to do that for two reasons, the first and more important one is that you can see what are the dependencies of the class instantly, the second and not that important, IMO, is performance (see https://veewee.github.io/blog/optimizing-php-performance-by-fq-function-calls/)